grepos/app/src/main/java/net/boondockradio/grepos/dto/Item.java

```diff
@@ -0,0 +1,1459 @@
+
+package net.boondockradio.grepos.dto;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Item {
```

Comment:
We try to avoid naming things `item`.
It is difficult to understand what an in could be.
Better to give it a meaningful name.

In this case I think this is a `Repository` dto.
And its parent would be `Repositories` dto.

The `Repositories` dto contains a list of `Repository` items